aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -384,6 +374,153 @@ Tensor& _clamp_min_out_cpu(Tensor& result, const Tensor& self, Scalar min) {
   return result;
 }
 
+Tensor clamp_with_tensors(const Tensor& self, const Tensor& min, const Tensor& max) {
+  Tensor result = at::empty({0}, self.options());
+  return at::clamp_with_tensors_out(result, self, min, max);
+}
+
+Tensor clamp_with_tensors_max(const Tensor& self, const Tensor& max) {
+  Tensor result = at::empty({0}, self.options());
+  return at::clamp_with_tensors_max_out(result, self, max);
+}
+
+Tensor clamp_with_tensors_min(const Tensor& self, const Tensor& min) {
+  Tensor result = at::empty({0}, self.options());
+  return at::clamp_with_tensors_min_out(result, self, min);
+}
+
+Tensor& clamp_with_tensors_(Tensor& self, const Tensor& min, const Tensor& max) {
+  return at::clamp_with_tensors_out(self, self, min, max);
+}
+
+Tensor& clamp_with_tensors_out(
+    Tensor& result,
+    const Tensor& self,
+    const Tensor& min,
+    const Tensor& max) {
+  TORCH_CHECK(!self.is_complex(), "clamp is not yet implemented for complex tensors.");
```

Comment:
Would you do me a favor and change these outdated "clamp is not yet implemented for complex tensors" to "clamp does not support complex inputs" ? The "yet" in the old warning (which I wrote, by the way) suggests clamp will eventually work on complex inputs, but we've since decided not to allow it.

Reply:
Sure, will do that.